rust/src/dataset/scanner.rs

```diff
@@ -769,7 +769,7 @@ mod test {
 
         let dataset = Dataset::open(path).await.unwrap();
         let mut scanner = dataset.scan();
-        scanner.limit(2, Some(19)).unwrap();
+        scanner.limit(Some(2), Some(19)).unwrap();
```

Comment:
what does it mean if we pass `limit(None, Some(19)`?

Reply:
Skip the first 19 rows and return everything after.